### PR TITLE
Disable monitoring by default

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @lance @osmman @ppecka

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,7 +49,7 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.33.0
+OPERATOR_SDK_VERSION ?= v1.32.0
```
Comment (lance) [inline Makefile:52]: @Gregory-Pereira is this an intentional downgrade?

**Review by lance — APPROVED** [review]
/lgtm

File: Makefile (hunk 1 above)
Comment (lance) [reply Makefile:52]: Answered in slack, "yes"

**Review by osmman — APPROVED** [review]

**Pull request merged**